recipes/pyopenms/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "pyopenms" %}
+{% set version = "2.9.1" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+{% if 'dev' in version %}
+source:
+  git_url: https://github.com/OpenMS/OpenMS.git
+  git_rev: develop
+  git_depth: 1 # (Defaults to -1/not shallow)
+{% else %}
+source:
+  url: https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/{{ version }}/OpenMS-{{ version }}-src.tar.gz
+  sha256: 2d186f35fe42573a19283605ccfb491ca6da585880fa3725a013975df26bc008
+{% endif %}
+
+build:
+  skip: True  # [py2k]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+  host:
+    - libopenms {{ version }}
+    - llvm-openmp  # [osx]
+    - libsvm
+    - xerces-c
+    - boost-cpp
+    - qt
```

Comment:
If you aren't linking to qt, which i doubt you are in a python package, just remove this. Let the c package drag this in if it needs to.